DQM/L1TMonitor/src/L1TStage2EMTF.cc

```diff
@@ -569,15 +608,22 @@ void L1TStage2EMTF::analyze(const edm::Event& e, const edm::EventSetup& c) {
     emtfTrackMode->Fill(mode);
     emtfTrackQuality->Fill(quality);
     emtfTrackQualityVsMode->Fill(mode, quality);
-    
-    // Only plot if there are <= 1 neighbor hits in the track to avoid spikes at sector boundaries
-    if (modeNeighbor < 2 || modeNeighbor == 4 || modeNeighbor == 8) {
-      emtfTrackPhi->Fill(phi_glob_rad);
-      if (quality >= 12) {
+    RPCvsEMTFTrackMode->Fill(mode, modeRPC);
+    emtfTrackPhi->Fill(phi_glob_rad);
+
+    if (quality >= 12) {
+        emtfTrackPtHighQuality->Fill(Track->Pt());
+        emtfTrackEtaHighQuality->Fill(eta);
         emtfTrackPhiHighQuality->Fill(phi_glob_rad);
-      }
+        emtfTrackOccupancyHighQuality->Fill(eta, phi_glob_rad);
+        if (Track->Pt()>=22) {
```

Comment:
sorry, would not be better to have the definition of a constant somewhere, isnetad of magic numbers popping-up in the middle of the code?

Reply:
I have added a second commit with int definition for track quality and track pt thresholds.